docs/guides/deployment/digitalocean.rst

```diff
@@ -0,0 +1,105 @@
+.. _ref_guide_deployment_digitalocean:
+
+===============
+On DigitalOcean
+===============
+
+In this guide we show how to deploy EdgeDB to DigitalOcean using a DigitalOcean
+managed PostgreSQL database as the backend.
+
+
+Prerequisites
+=============
+
+* DigitalOcean account
+* ``doctl`` CLI (`install <doclt-install_>`_)
+
+.. _doctl-install: https://docs.digitalocean.com/reference/doctl/how-to/install/
+
+
+Create a managed PostgreSQL instance
+====================================
+
+Note the URI that is printed in the output from this command. It will be used
```

Comment:
Is there a way to obtain the URI with a CLI command?  Ideally, the guide should be completely copy-pastable to reduce space for user error.

Reply:
Most `doctl` commands provide a way to get single values likes, but the `doctl database create` command does not have that functionality. The nicest way would to use the `--output json` option with `jq` but that adds another dependency. Other wise we could do some ugly text parsing with grep & cut, sed, awk etc. Not sure which is better.

There also does not seem to be a programmatic way to get the URI for a database is all you  know is the name. You need the id to use `doctl database get`.

Reply:
`jq` isn't an unreasonable thing to rely on, it's widely available.